### PR TITLE
Babel setup

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

## Show me
_Provide screenshots/animated gifs/videos if necessary._

# REMEMBER: Attach this PR to the Trello card

### CODE REVIEW

File: babel.config.cjs [hunk 1]
```diff
@@ -1,6 +1,5 @@
 module.exports = {
   presets: [
-    ["@babel/preset-env", { targets: { node: "current" } }],
-    "@babel/preset-typescript",
```
Comment (sanajaved7) [inline babel.config.cjs:4]: @oluoluoxenfree if we're using typescript, should we still keep `@babel/preset-typescript` here?

Comment (oluoluoxenfree) [reply babel.config.cjs:4]: I thought since we're only changing the floating-ui package that it wasn't needed?

Comment (sanajaved7) [reply babel.config.cjs:4]: Oh but all of our logic, outside of floating-ui, will be in TS/JS so that would need to be transformed too, right?

Comment (oluoluoxenfree) [reply babel.config.cjs:4]: @sanajaved7 i thought that esbuild was handling that, or what does it do? *goes to check*

Comment (sanajaved7) [reply babel.config.cjs:4]: btw @oluoluoxenfree want to add that bable/esbuild plugin as part of this branch? I think after that we could go ahead and merge. What do you think?

**Comment by sanajaved7** [comment]
[Refine build process](https://trello.com/c/sTQjtl80/39-refine-build-process)

**Comment by oluoluoxenfree** [comment]
I thought that's what esbuild did?

**Pull request closed without merging**